docs/examples/composer_app/composer.json

```diff
@@ -0,0 +1,6 @@
+{
+    "require": {
+        "monolog/monolog": "1.3",
+        "marcelog/pami": "dev-master"
```

Comment:
I'm not sure about that. I've tested it by specifying `dev-no_more_pear` and `dev-master` and it worked.. also it uses dev-branch here: https://getcomposer.org/doc/05-repositories.md#using-private-repositories